src/Liquidator.sol

```diff
@@ -51,7 +51,6 @@ contract Liquidator is Owned, ILiquidator {
 
     // Struct with additional information about the auction of a specific Account.
     struct AuctionInformation {
-        address originalOwner; // The address of the original owner of the Account.
```

Comment:
I think better to keep this, it cuts one call to account in the happy flow. And it is good data point to query later on

Reply:
What do you mean with "good data point to query later on"?
Changes of ownership emit events as well.